### DISCUSSION TITLE
Guz' Development

### DISCUSSION THREAD

**Discussion by lumbar527** [OcelotWalrus/Cromha-Expansion-plugin#124] [General]
This is a place to make suggestions/review the creation of a new alien species: the Guz'Pas!

<details><summary>Document</summary>
<p>

### Home

They are from the Milky Way, but they found a new home further away. (Might remove the further away home part)
Have to figure out a place in the Milky Way, though.

### Technology

They use heavy artillery weaponry with some ammo. (A big turret for a smaller ship might hold 7 rounds)

They mostly use gunboat-size ships.

They're probably tier 2 because their ships are weak health-wize, more like a little stronger than human, but their weaponry is very powerful. They also swarm and have large fleets.

They also have interesting shields. Their shields have a several-second recharge delay, but that does not make as much of a difference. They actually have fairly strong shields compared to their light hull, but they always let 25% of the damage through. They do not need as much energy, though. Now, their shields use is in its resistance to effects: ion, heat, disruption, etc.

### Appearance

They look like bipedal tortoises that are a little taller than humans, but their arms are more like human arms. They have skinnier arms with hands, but they cannot bend them as much as humans can, because of a partial shell covering the elbow and along that side of the arm. These arms are also very heavy, and they can strike you dead in a single blow.

### History

The Guz'Pas are a relatively new species, only about 6,000 years old (at game start). They developed spaceflight fairly quickly when they discovered their star was going to supernova (B.C. 1,000; 2,000 years old). Only a hundred years later, they met another race, and there was a great war. It lasted 300 years, leaving at a draw when neither could fight more. (the other species may have disappeared by now, though) Even though the Guz'Pas had not made weaponry before the war, they still managed to last that long using big missiles they could scrap together.

By this time, they had learned how to make powerful missiles cheaply. This became their primary defense, and slowly they moved onto the offense. By A.D. 1, they were already spreading out and had made contact and wars with several other aliens. (Maybe they fought the Sheragi?) Then the Great Attack happened, leaving them shrinking.

The Great Attack happened around A.D. 100, when they believed they were attacked by many strange enemies. Several of their major stars went supernova, and ships mysteriously disappeared. (Was it an attack, or was it some anomaly or something that caused this?) After a few hundred more years, they were back to a few systems.

Then they found the wormhole, and went through it. There they found systems and planets to colonize, a secret territory. But while they were colonizing, they found a weaker race that had only a few systems and no hyper/jump drives. There was a war with them that lasted 17 years, ending in the destruction of the weaker race.

After that, they lived in peace but did not expand much, until humans began to expand. Then they destroyed some convoys, but then abandoned some of their systems on the edge. There was another wormhole that connected their space to what would become human space, though the gap between their systems was small enough to use Jump Drives on.

Around A.D. 2,600, the wormhole disappeared. They were now isolated from all other races.

In the time between them leaving and the player starting, the design their shields and begin to build larger transport/cargo ships, but still new warships. (the transport/cargo ships are more medium warship to a little bigger sized.

</p>
</details>

<details><summary>Outfits TO-DO</summary>
<p>

- [x] ~Drives (maybe just one hyper drive or multiple hyper drives)~
- [x] Solar Collection systems and ramscoops
- [x] Storage power systems
- [x] Compact engines (5 engine class that all have reverse, steering and forward thrust integrated)
- [x] Reactor to generate energy
- [ ] Smaller outfits to generate energy similar to humans technologies
- [x] Scanners
- [ ] Ameliorating Cells?
- [x] ~Hand to Hand/Defense outfits~

all other types of outfits will be vanilla ones from the humans.

</p>
</details>

<details><summary>Technologies</summary>
<p>

They do not have outfits cooling systems because they already have `active cooling` attributes to their ships depending on the ship heating. And the `active cooling` shall not use energy.

Ships also have `energy capacity` attribute integrated to the ship. Maybe we will decide to add more attributes integrated.

Their ships all have `never disabled` attribute, without exception.

</p>
</details>

<details><summary>Downloadable Files</summary>
<p>

Separate Guz'Pas plugin, though only has shields currently:
[Guz'Pas.zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12207383/Guz.Pas.zip)
Assets:
[guz'pas shield medium.blend.zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12207392/guz.pas.shield.medium.blend.zip)
[guz'pas shield small.blend.zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12207393/guz.pas.shield.small.blend.zip)
[guz'pas shield tiny.blend.zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12207394/guz.pas.shield.tiny.blend.zip)
[final (unscaled).zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12207395/final.unscaled.zip)
Sorry, the ship is too large to upload here.

</p>
</details>

**Comment by OcelotWalrus** [comment]
Looks great! But why do you create a discussion on this repo if it's an other plugin where the Guz'Pas are developed?

**Reply by lumbar527** [reply]
Because I want to do some separate work to keep everything organized. Also, I'm using the editor and don't want other random stuff to get messed up. Once I have some more stuff I'll commit it.

**Reply by lumbar527** [reply]
But if you think I should just develop on the main repo, then I'll move files over.

**Reply by OcelotWalrus** [reply]
No that's fine if you work on your own repository.

**Reply by lumbar527** [reply]
There's no repo; just some files on my computer.

**Comment by OcelotWalrus** [comment]
I have a question:

What are the Guz'Pas inter-species relations?

**Reply by lumbar527** [reply]
I don't think they've met many other species, but the are a little warlike so in most cases the would attack. (The Lumbarian Empire will have to work on an a treaty with them, not just finding them and sticking them in their enemies faces.)

**Reply by OcelotWalrus** [reply]
Okay. So the Guz'Pas will be an emerging species or a young under-developed species? Or a developed species?

**Reply by OcelotWalrus** [reply]
I think they should be an emerging species.

**Reply by lumbar527** [reply]
I was thinking they might be more like a young species - see **History** under "Documents" tab.
If you want to suggest something, though, feel free to write out an alternative and then they can be compared and revised into one.

**Reply by OcelotWalrus** [reply]
yes. Could you make the Guz'Pas wiki page for their faction please?

**Reply by lumbar527** [reply]
Sure, though it might be missing a few things.

**Reply by lumbar527** [reply]
Done, for now.

**Reply by OcelotWalrus** [reply]
Nice, thanks!

**Comment by OcelotWalrus** [comment]
I've got an idea for the Guz'Pas:

### The civilizations
The Guz' species will be actually divided in 3 factions (kinda like the Gegno):

The Guz'Pas (the military-base-like faction), the Guz'Huk (a commercial-based-like faction) and the Guz'Kah (a pirate-like faction).

All 3 factions are not hostile to each other but are not in good relations.

The Guz'Kah often sends raids fleet to samll Guz'Huk freighters fleets and the Guz'Pas don't mind that much about the other Guz civilizations but is interested in upgrading their technologies and exploring the space, while the Guz'Huk search for commercial hyperspace routes and abundant primary materials ad the Guz'Kah are kinda the Guz' Pirates.

But they will still all shares the same technologies.

### Their space region
The Guz' space could be a very northern region that cannot be acceded with any drives but only by a wormhole located in an uninhabited system (need to search for a system or just create a new one especially for that). The Guz' would be the only species in their space.

**Reply by lumbar527** [reply]
Great, I like that. I'll move some systems I already had and add a new one to the space between the core and rim. Then, north but out of jump range from the Wanderers, there they would have their actual systems.

**Reply by OcelotWalrus** [reply]
I think you should add the Guz' to the main branch on this repository.

**Reply by lumbar527** [reply]
Sure, I'll put that in.

**Reply by OcelotWalrus** [reply]
Thanks!

**Reply by lumbar527** [reply]
Sorry, it's having pushing problems. At least one of my files is too big, I think.

Edit: The `guz'pas chisel.blend` file is like, 995.1 MB. How did I make it so big????? Compression can only get it to 255.9 MB.

**Reply by OcelotWalrus** [reply]
You can run `git config --global http.postBuffer 524288000` before pushing. It will raise the maximum size per file limit while pushing.

**Reply by lumbar527** [reply]
Doesn't work. Is there a way I could send you the files without using GitHub and you could upload them?

**Comment by OcelotWalrus** [comment]
Why did you made that many files?
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/b987a0b6-a3db-49b4-8d7a-37cd9ca92ddd)

**Reply by OcelotWalrus** [reply]
And by the way? Could you use GIMP if possible for next times? So anyone can open the files.

**Reply by lumbar527** [reply]
I guess I could download GIMP. As for the file count, I accidentally made the ship transparent and then had to create a new file to fix that.

Edit: GIMP is confusing.

**Reply by lumbar527** [reply]
I have no idea how to use GIMP. OK, I can open an image and draw some lines, but selecting works badly and I don't really understand much. Pixelmator Pro works much better. (Maybe I can find a file converter, to turn .pxd into whatever GIMP uses)

**Comment by OcelotWalrus** [comment]
Could you update the wiki page of the Guz' to add the different factions thing?

_[Check this](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/wiki/Governments-&-Factions#tier-2)_

**Comment by OcelotWalrus** [comment]
I found some interesting images for possible Guz' outfits:

https://github.com/OcelotWalrus/Cromha-Expansion-plugin/tree/main/images/outfit/new

**Reply by lumbar527** [reply]
Interesting, but no rifle. They don't use extra hand-held weaponry. The tractor would be interesting, but I'm not sure.

**Reply by OcelotWalrus** [reply]
The `guz reasoning node.png` could be a `Guz Control Transceiver`, basically a multi-usage scanner.

The `guz license.png` could be a `Guz Traductor`, kinda like the Wanderer Translator that you will get while the Guz' storyline missions.

**Reply by lumbar527** [reply]
I like that!

**Comment by OcelotWalrus** [comment]
About the Guz' Tractor, it could be a beam weapon similar to the Rebels Grav-Turret. I'm gonna code that right now.

**Reply by lumbar527** [reply]
It's a recent breakthrough, one of their few non-missile weapons. (They probably have some anti-missile but it might be crude)

**Reply by OcelotWalrus** [reply]
I'm also creating placeholders for their licenses as the same time.

**Reply by OcelotWalrus** [reply]
> (They probably have some anti-missile but it might be crude)

We could say that since they are a young species, they did not developed/found anti-missiles technologies. Or maybe they did not searched about anti-missiles because they didn't had any use case?

**Reply by OcelotWalrus** [reply]
I finished: e89f87d014e68b839e7906ffbb6370c9dcb1ab44

**Reply by lumbar527** [reply]
If the Guz'Kah raid the Guz'Huk then the Guz'Huk want AM.

**Comment by OcelotWalrus** [comment]
About the Guz'Chisel, what size should it be? I though about something like 140x200 pixels.

140 times 200 is the size of the Bastion.

**Reply by OcelotWalrus** [reply]
Or maybe the Chisel could be like a `Transport` class or a `Fighter`/`Drone` class?

Because since the design is simple, it looks more like a Fighter/Drone ship.

**Reply by lumbar527** [reply]
About 200x200.

**Reply by OcelotWalrus** [reply]
Okay. I'm gonna test 190x190 first.

**Reply by lumbar527** [reply]
Won't that warp the image?

**Reply by OcelotWalrus** [reply]
No, since 2000 (size of the image) is a dividable bye 10 or 2 and 190 is dividable by these numbers too.

**Reply by OcelotWalrus** [reply]
I'm gonna also code the ship as the same time.

**Reply by OcelotWalrus** [reply]
Is it okay if I make it a Utility-class ship?

**Reply by lumbar527** [reply]
It's a Light Warship/Gunboat.

**Reply by OcelotWalrus** [reply]
Okay. I put 1 turret mount, 2 gun ports and 2 fighter bays.

**Comment by OcelotWalrus** [comment]
I added the Guz'Chisel with dd21baf.

**Reply by OcelotWalrus** [reply]
Here is how it looks in-game: (asteroid for size comparison)
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/b00f579a-dc50-4a91-add0-8f76b130f05f)

**Comment by OcelotWalrus** [comment]
I got an idea: what if their ships all have `never disabled` attributes, and says that they didn't develop the automatic disabling system?

**Reply by lumbar527** [reply]
I don't think disabling is a system; it's that your ship is too damaged to operate.

**Reply by OcelotWalrus** [reply]
No, it's actually a system that automatically disable your ship for safety when its hull is too low.

**Reply by lumbar527** [reply]
Interesting...sure!

**Reply by OcelotWalrus** [reply]
I just added it.

**Comment by lumbar527** [comment]
Would it be OK if I handled the data more? Like did the making of the actual ship definitions and stuff.

**Reply by OcelotWalrus** [reply]
Sure, you can make the chisel and any other ship, for the chisel I just added hardpoints and guz shields but nothing more.

**Reply by lumbar527** [reply]
I tweaked values and removed the bays, so far. I still need to make weapons! (I'll get to that soon)

**Reply by OcelotWalrus** [reply]
Maybe you can install the Tractor turret for now.

But before creating any weapon, we need to create basic systems outfits like power storage, energy generators, drives, and other basic systems.

**Reply by OcelotWalrus** [reply]
I'm gonna make a list of all outfits to create.

**Reply by lumbar527** [reply]
Thanks! (I can help with names - Glusk Batteries, Hkthor Generator)
They'll use regular Hyperdrives because I plan to PR to vanilla.

**Reply by OcelotWalrus** [reply]
Sure. Check the TO-DO list on the [top](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/discussions/124#top) of the conversation.

**Reply by lumbar527** [reply]
Thanks for making the list!

**Comment by OcelotWalrus** [comment]
Is it okay if I create the Guz' Scanning Systems?

**Reply by lumbar527** [reply]
Sure, want some possible names?

<details><summary>Potential Names</summary>
<p>

Sklorai Scanner, Kroth Scanner, Holk Scanner
Written in order from larges to smallest. (Fell free to remove any of these)

</p>
</details>

**Reply by OcelotWalrus** [reply]
Thanks! I'm gonna use all of these three names.

**Reply by OcelotWalrus** [reply]
I'm using these textures:
Tiny
![guz control transceiver](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/d0af732c-c8bb-4c7e-ac30-dc43f3aa3cda)
Small:
![salvage scanner](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/c93998b3-5d3e-43cb-b7af-68c6647cd00b)
Large:
![value detector](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/fb7c0823-7368-4a7a-af4f-83fac4d82642)

**Reply by lumbar527** [reply]
Cool! I especially like the tiny one!

**Comment by OcelotWalrus** [comment]
> If the Guz'Kah raid the Guz'Huk then the Guz'Huk want AM.

What do you mean by 'want AM'?

**Reply by lumbar527** [reply]
The Guz'Huk would want some way to defend themselves against the powerful missiles, so they might develop AM (anti-missile) or radar jamming.

**Reply by OcelotWalrus** [reply]
Oh I see, maybe.

**Comment by OcelotWalrus** [comment]
> Doesn't work. Is there a way I could send you the files without using GitHub and you could upload them?

Sure, just send me them and I will upload them. I just need the .blend file.

**Reply by lumbar527** [reply]
How would I send them to you?

**Reply by OcelotWalrus** [reply]
I don't know, maybe through ~Good~ _Google_ Drive, Dropbox or something like that?

**Reply by lumbar527** [reply]
It's here, I think: <link removed so I can be sure it's just Cromha and I who can edit>

**Comment by lumbar527** [comment]
Digging through old projects, what do you think of these engine for the Guz?
![diamond steering large](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/65305f7d-e4c5-46da-b063-4c5540250b5d)
![diamond steering medium](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/13d52837-1ba3-438f-b3f7-6d9a5a114fda)
![diamond steering small](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/be8d2927-1b38-431b-9506-fac677bb543b)
![diamond steering tiny](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/50cada53-da48-4b00-a815-e430d11e6eff)
![diamond thruster medium](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/d87f80c2-3e07-4630-907b-cdc8f93b7b24)![diamond thruster tiny](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/d03d4644-fc0d-4794-8ce5-842335fdb12e)
![diamond thruster small](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/316cefef-ff79-45ac-9ad0-e4a43bd6f626)
![diamond thruster large](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/195e271c-39a2-4793-bd2d-0ee2cd559403)

**Reply by OcelotWalrus** [reply]
I think the original shape is good. I could make good looking engines with these as models I think.

**Reply by lumbar527** [reply]
Thanks!

**Comment by lumbar527** [comment]
Also some potential fuel cells:
![diamond battery small](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/412c00d5-b5ce-4a4b-a3b0-f68cdce31a5d)
![diamond battery large](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/6a13e89e-a811-460a-beab-0722613b528c)
![diamond battery medium](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/b2b1ab5c-e279-4366-bd1a-993acbe8a4a5)

**Reply by OcelotWalrus** [reply]
The shapes looks good. It will just need some in-depth texturing. It could also be batteries.

I also thought, maybe the Guz' have integrated energy storage in all their generation systems? Since I added energy storage to all existing Guz' power generators.

**Reply by lumbar527** [reply]
Sure!

**Comment by OcelotWalrus** [comment]
> How would I send them to you?

You could directly upload your file on the github repository. It will be easier.
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/41b86202-9e84-4a2e-a9f6-28bf8cb39da8)

**Reply by lumbar527** [reply]
That has and even smaller file limit: 25MB.

**Reply by lumbar527** [reply]
Sorry, I think I gave you access to it now.

**Reply by OcelotWalrus** [reply]
Thanks!

**Comment by OcelotWalrus** [comment]
Could you send me all the blend files of the potential fuel cells and potential engines? I would like to create the sprites then you will be able to do all the coding :) I really want the plugin to have good quality textures/images.

**Reply by lumbar527** [reply]
[diamond engines:fuel cells.zip](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/files/12243237/diamond.engines.fuel.cells.zip)
Quick note: apparently the suggested fuel cells were supposed to be batteries, so that's what their files are called.

**Reply by OcelotWalrus** [reply]
Okay, but I will make them fuel cells, is that okay?

**Reply by OcelotWalrus** [reply]
Just, could you give me names for the fuel cells and the engines?

**Reply by lumbar527** [reply]
> Okay, but I will make them fuel cells, is that okay?

Yes, I prefer that!

Names:
Gloshk, Gloiak, and Glzak are the fuel cells (smallest to largest)
Kroth, Vlazk, Akrosh, and Zarkier are the engines (smallest to largest)
Some engines may not be needed; they use small ships.

**Reply by OcelotWalrus** [reply]
Thanks!

I think 4 engines series is actually good.

**Reply by lumbar527** [reply]
You're welcome! Do you like how I used meatballs for the engines?

**Reply by OcelotWalrus** [reply]
What do you mean by 'meatballs'?

**Reply by lumbar527** [reply]
<img width="769" alt="image" src="https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/1bb4b15a-e21d-4e2b-9162-6c137dcdacf2">
Those Blender things.

**Reply by OcelotWalrus** [reply]
Oh I see. You wrote meatballs bro.

**Reply by lumbar527** [reply]
I thought it was OK because I used multiple meatball.

**Reply by OcelotWalrus** [reply]
Sure.

**Comment by OcelotWalrus** [comment]
How did you made your blender model 949 mega bytes?

**Reply by OcelotWalrus** [reply]
I'll try to compress it.

**Reply by lumbar527** [reply]
I have no idea. It also gets slow when I view shading stuff.

**Reply by lumbar527** [reply]
Thanks!

**Comment by OcelotWalrus** [comment]
> I got these images (taken from 1010todd plugins): ![nuke 9 turret](https://user-images.githubusercontent.com/87318892/258212136-140eb5ad-c54c-491b-b041-b4ceb8982cba.png) ![nuke 9 turret hp](https://user-images.githubusercontent.com/87318892/258212147-a202fd56-0309-4ab8-9a63-431b844f0c24.png) ![nuke gat](https://user-images.githubusercontent.com/87318892/258212092-63dea157-ffc1-406f-96e4-6dda48530d8c.png)

Maybe instead of giving the nukes to the Lumbarian Empire, why not give it to the Guz', because they are a low-tier species and their principal attribute is nuclear energy.

**Reply by OcelotWalrus** [reply]
And since the Guz' are specialized in ballistic weapons, it could be good to give them the nuke.

**Reply by lumbar527** [reply]
I don't know if the Guz have developed nukes. I hadn't actually considered that when talking about how they use artillery missiles. But if you think they should have nukes, then we can do that.

**Comment by OcelotWalrus** [comment]
About the Guz' Engines,

I finished the make the outfit thumbnail of the first 3 tier thrusters (there are 5 in total):
![tier 1 thruster](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/06603589-eba1-4776-8e23-de2133815866)
![tier 2 thruster](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/8c12578f-c9c4-4a02-9e11-bc9ee989cb84)
![tier 3 thruster](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/c9f0a1a2-9988-4926-a571-f4a8eca9b13a)

**Reply by OcelotWalrus** [reply]
I think the tier 3 is too bright compared to the other.

Should i lower the tier 3 brightness or make the tier 1 & 2 brightness higher?

**Reply by lumbar527** [reply]
I think lowering the tier 3 brightness would be better.

I really like what you did with this!

**Reply by OcelotWalrus** [reply]
Thanks! They will also have integrated afterburning that have the same thrust than the basic thrust.

**Reply by lumbar527** [reply]
That works!

**Comment by OcelotWalrus** [comment]
I've come up for something about Guz' technologies:

They do not have outfits cooling systems because they already have `active cooling` attributes to their ships depending on the ship heating. And the `active cooling` does not use energy.

Ships also have `energy capacity` attribute integrated to the ship. Maybe we will decide to add more attributes integrated.

Their ships all have `never disabled` attribute, without exception.

I wrote this on the first discussion comment under 'technologies' so we remember.

**Reply by lumbar527** [reply]
Interesting!

**Comment by OcelotWalrus** [comment]
![genesis repair bot](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/b17d5321-8579-4cc8-9d3f-d27109ba3f5c)

Maybe with this texture we could make it so it's a Guz' outfit that repairs the hull but it has turrets attributes (take a turret mount and has an hardpoint), it could be an interesting new concept.

**Reply by lumbar527** [reply]
That sounds cool!  We need to think of why it needs to be a turret, though. Maybe it's actually not a hull repair system; it's a shield thing that actually gives the ships extra shields! (For balance, it would require a special attribute)

**Reply by OcelotWalrus** [reply]
We could say that the hardpoint thing is a repairing hull bot fabricator. So the hardpoint would be animated.

**Reply by lumbar527** [reply]
That sound great, and probably would be better for balance.

**Reply by OcelotWalrus** [reply]
Yes.

**Comment by OcelotWalrus** [comment]
> That sounds cool! We need to think of why it needs to be a turret, though. Maybe it's actually not a hull repair system; it's a shield thing that actually gives the ships extra shields! (For balance, it would require a special attribute)

I think we would use that for an other outfit for the Guz' (that you can code right now if you want). This outfit will be sort of a "Guz'" ameliorating cell that has shields multiplier attributes but also a custom `guz ameliorating capacity` attribute set to -1. So Guz' Ameliorating Cells will be installable only on Guz' ship. So all Guz' ships will have a defined amount of `guz ameliorating capacity` attribute.

**Comment by lumbar527** [comment]
I think I made a possible Guz Artillery Turret (9f909b18048293bcda0cde90d20f927293695d78)

**Reply by OcelotWalrus** [reply]
Nice! I'll test it on the editor and maybe propose some changes.

**Reply by OcelotWalrus** [reply]
I suppose the textures are placeholders?

**Reply by lumbar527** [reply]
I used the Genesis Flax Turret for the turret, but the missile definitely is a placeholder image.

**Reply by OcelotWalrus** [reply]
I'll try to find/create textures

**Reply by lumbar527** [reply]
Thanks!

**Comment by OcelotWalrus** [comment]
Just, remember it's Guz' and not Guz in text and outfit names.

**Reply by lumbar527** [reply]
I'm confused about that. Why?

**Reply by OcelotWalrus** [reply]
That's how it's named am I right?

**Reply by OcelotWalrus** [reply]
Guz'Pas, Guz'Huk, Guz'Kah

**Reply by lumbar527** [reply]
Yes, but the `'` is for separation inside the words, I thought? And actually maybe all of them together should be the Guz Agzizigon?

**Reply by OcelotWalrus** [reply]
I'd prefer it to be Guz' and not Guz, but if it's a problem for you, it'll be Guz.

**Reply by lumbar527** [reply]
It's fine; they can be the Guz'.

**Comment by OcelotWalrus** [comment]
I fully finished the Guz' Thrusters (thumbnails and definition). I know have to work on the steering.
Thumbnails:
![tier 1 thruster guz](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/eae411cc-3662-4d40-a26f-579f4f7ff8cf)
![tier 2 thruster guz](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/5c094d9e-e856-464c-b429-0c73443aab4f)
![tier 3 thruster guz](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/f64956a0-d44e-490d-a46f-9556d21fbb83)
![tier 4 thruster guz](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/10ef9789-ab7f-4232-9140-da2f23bd4aba)
![tier 5 thruster guz](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/304a5792-f2b2-4759-836b-d394a9ffeba1)

**Reply by lumbar527** [reply]
![image](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/125756761/e95e78c6-82f4-468e-b6b0-19fc4b018710)
Wow...that's massive!

**Reply by OcelotWalrus** [reply]
I know, it's the tier 5 thruster, the final tier.

**Reply by lumbar527** [reply]
Now the Guz' need some giant endgame ships to use that.

**Comment by OcelotWalrus** [comment]
I've made a new ship model for the Guz': the Chasef, I only made the sprites to you can code the ship. The two gun ports are supposed to be gatling gun port. I will develop a gatling gun for the Guz' soon.
![chasef](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/2c68d957-6feb-4bed-a5b5-e7f69df6c23f)
![chasef](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/assets/87318892/d3d90977-0ba4-4636-9a92-df7a9809e979)

You can propose a new name for the ship if you want.

**Reply by OcelotWalrus** [reply]
EDIT: I made a gatling gun for the Guz', it's the `Guz' Gatlin' Gun`.

**Reply by lumbar527** [reply]
Thanks!

**Reply by OcelotWalrus** [reply]
You're welcome!

**Comment by OcelotWalrus** [comment]
I have a question:

How you wish the Guz' ships look like, what are their design-type?

**Reply by lumbar527** [reply]
Similar to the one I made; mostly gunboat-sized, with a turret or two and a few gun ports, mostly for artillery missiles.

In shape, probably with a sticklike body and wing.

**Reply by OcelotWalrus** [reply]
Okay thanks!

**Comment by OcelotWalrus** [comment]
Just some news about the Guz', I will try to continue working on them! I finished the tier 1 to 3 sprites and they are coded to the game!

**Reply by OcelotWalrus** [reply]
I've added two new outfits for the Guz':
![image](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/assets/87318892/7224b9bf-d3f3-4aa9-8883-ea939413aac8)
It's a turret just so there is a hardpoint on the ship and so that it kind of balance the outfit, it does not fire anything at all.
![image](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/assets/87318892/35efd4fb-32d4-40d6-8469-8f310cbbd69a)

**Reply by OcelotWalrus** [reply]
Please tell me @lumbar527 if there is re-balance or buffing needed to be done on these outfits.

**Reply by lumbar527** [reply]
Seems fine. If you want you can just take over; I'm not good at continuing projects.

**Reply by OcelotWalrus** [reply]
Okay, so you're not interested anymore by the plugin?

**Reply by lumbar527** [reply]
No, I still think it's cool and I may actually start working more on it again.

**Reply by OcelotWalrus** [reply]
> No, I still think it's cool and I may actually start working more on it again.
Cool! Because I actually recently re-found interest about this plugin.

**Comment by OcelotWalrus** [comment]
Hey! Just want to let you know @lumbar527 that I began working on the Guz' storyline and first contact.

**Reply by OcelotWalrus** [reply]
I'll make a "prototype" commit soon for you to test.

**Comment by OcelotWalrus** [comment]
It took more time than expected, but I coded the Guz' First contact mission by commit 0dd752b6b701771a40fef0ac822e4102c23943e0!

If you want to test, just go to the Guz' Makk system and land on Guz'Ulz.
![image](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/assets/87318892/84741465-620a-4829-b879-462d445fbd70)

**Reply by OcelotWalrus** [reply]
I think we first need to fully finish Guz' technologies (ships & outfits) as well as their systems before continuing the storyline.

**Reply by OcelotWalrus** [reply]
P.-S.: I worked hard on the dialogs of the first contact.

**Comment by OcelotWalrus** [comment]
Thanks @MonkeyMatrix for contributing to the project!

**Comment by OcelotWalrus** [comment]
Just edited the Guz' wiki page
[https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/wiki/Guz'](https://github.com/Cromha-Plugins/Cromha-Expansion-plugin/wiki/Guz')

**Reply by lumbar527** [reply]
Nice!
